app/src/main/res/layout/fragment_user_login_credentials.xml

```diff
@@ -38,19 +38,42 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/username" />
 
-    <EditText
-        android:id="@+id/password"
-        style="@style/Input.Default"
-        android:layout_width="0dp"
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Merge this constraint layout into the parent constraint layout. Traditional layouts are inefficient.